src/Estimators/EstimatorManagerNew.cpp

```diff
@@ -123,9 +119,11 @@ void EstimatorManagerNew::setCommunicator(Communicate* c)
  */
 void EstimatorManagerNew::reset()
 {
+  //It's essential that this one is first, other code assumes weightInd always == 0
   weightInd = BlockProperties.add("BlockWeight");
+  assert(weightInd == 0);
   cpuInd    = BlockProperties.add("BlockCPU");
-  acceptInd = BlockProperties.add("AcceptRatio");
+  acceptRatioInd = BlockProperties.add("BlockAcceptRatio");
```

Comment:
Can we keep the name?

Reply:
yes for the property, the variable name did not express what this was.